examples/hot-reload/skaffold.yaml

```diff
@@ -32,7 +32,7 @@ profiles:
             DEBUG: 1
         sync:
           manual:
-          # Sync all the javascript files that are in the src folder
+          # Sync all the python files that are in the src folder
```

Comment:
Can you also make this change here:
https://github.com/GoogleContainerTools/skaffold/blob/d030bc1bb6f446591f300b2132c27f806d8036ee/integration/examples/hot-reload/skaffold.yaml#L35-L38

Reply:
Done!